Gosu/Window.hpp

```diff
@@ -111,8 +111,11 @@ namespace Gosu
         //! in your implementation.
         virtual void button_down(Gosu::Button);
         
-        //! Same as button_down. Called then the user releases a button.
+        //! Same as button_down. Called when the user releases a button.
```

Comment:
Just an typo I found - not related to the actual request but didn't want to make a seperate commit / PR